src/main/java/org/sagebionetworks/bridge/hibernate/HibernateAccountDao.java

```diff
@@ -254,6 +254,7 @@ AccountSummary unmarshallAccountSummary(HibernateAccount acct) {
         builder.withSynapseUserId(acct.getSynapseUserId());
         builder.withAttributes(acct.getAttributes());
         builder.withOrgMembership(acct.getOrgMembership());
+        builder.withNote(acct.getNote());
```

Comment:
Let's also add this to HibernateAccountDaoTest.unmarshallAccountSummarySuccess(), to make sure we are exercising this piece of code.

Reply:
The HibernateAccountDaoTest.unmarshallAccountSummarySuccess() test should be updated now to check that the note field is coming through in the Account Summary.